<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>675</width>
    <height>443</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="receive">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>190</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>681</width>
      <height>311</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>03-screen</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>10</y>
        <width>261</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>follower condition</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>44</y>
        <width>291</width>
        <height>201</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="4" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>元/度</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="rate"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>工作风速</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="wind_stren"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>使用费用</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="price"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>目标温度</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="aim_tem"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>当前温度</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="current_tem"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>385</x>
        <y>50</y>
        <width>281</width>
        <height>111</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_9">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>host</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEdit_host"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_10">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>port</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="lineEdit_port"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="connect_mes">
         <property name="text">
          <string>准备连接</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QPushButton" name="connect">
         <property name="text">
          <string>连接</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label_windicon">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>190</y>
        <width>54</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_modelicon">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>190</y>
        <width>54</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <zorder>layoutWidget</zorder>
     <zorder>label</zorder>
     <zorder>gridLayoutWidget_2</zorder>
     <zorder>label_windicon</zorder>
     <zorder>label_modelicon</zorder>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>03-controller</string>
     </attribute>
     <widget class="QWidget" name="gridLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>60</y>
        <width>591</width>
        <height>131</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="2">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>调节风速</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>调节温度</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="tem_change"/>
       </item>
       <item row="0" column="3">
        <widget class="QComboBox" name="wind_change">
         <item>
          <property name="text">
           <string>低</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>中</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>高</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="start_button">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>310</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/off</normaloff>:/off</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>75</width>
      <height>75</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
